hil/test_common.py

```diff
@@ -437,7 +438,21 @@ def obmd_cfg():
         be used as a fixture, but not declared here as such; individual
         modules should declare it as a fixture.
         """
-        portno = 8833
+        # Make the port number a function of which worker we're running on,
+        # so parallel tests don't try to use the same port. We derive the
+        # port number by stripping out all non-digit characters from the
+        # worker id, and then adding the result to a base. If the worker id
+        # isn't set, we're probably not running in parallel, so just use
+        # the base.
+        portno = 8800
+        worker = os.getenv('PYTEST_XDIST_WORKER')
```

Comment:
Out of curiosity, how did you figure out that this variable refers to gw0/1/2.. in the output? `plugins: requests-mock-1.5.0, xdist-1.22.2, forked-0.2, cov-2.5.1
gw0 C / gw1 C / gw2 C / gw3 C` Could you point me to the documentation because I couldn't find